### ISSUE TITLE
Provide a way to override `alternativeAdvertisingNameEnabled`

### ISSUE THREAD

**Issue by townofdon** [Pilloxa/react-native-nordic-dfu#90]
Hello!

Awesome lib. I have a quick question:

The library `IOS-Pods-DFU-Library` has this option to turn off `alternativeAdvertisingNameEnabled`:
- https://github.com/NordicSemiconductor/IOS-Pods-DFU-Library/blob/master/iOSDFULibrary/Classes/Implementation/DFUServiceInitiator.swift#L191

Would it be possible to add this as an option when calling `NordicDFU.startDFU`?

If I'm reading this correctly, it would be easy enough to [set the instantiated `DFUServiceInitiator ` class properties here](https://github.com/Pilloxa/react-native-nordic-dfu/blob/5d7b6ffe46e9f0c7671e52b5d5eb99d59a227363/ios/RNNordicDfu.m#L227)?

I'm curious if the following would work? If so I'll submit a PR:

```
RCT_EXPORT_METHOD(startDFU:(NSString *)deviceAddress
                  deviceName:(NSString *)deviceName
                  filePath:(NSString *)filePath
                  resolver:(RCTPromiseResolveBlock)resolve
                  rejecter:(RCTPromiseRejectBlock)reject
                  // new param:
                  alternativeAdvertisingNameEnabled:(BOOL *)alternativeAdvertisingNameEnabled)
{
  ...
  initiator.alternativeAdvertisingNameEnabled = alternativeAdvertisingNameEnabled
  ...
}
```

Forgive me if any of my syntax is off - I'm new to Objective C. :)

**Comment by townofdon** [comment]
Submitted PR: https://github.com/Pilloxa/react-native-nordic-dfu/pull/91

**Comment by townofdon** [comment]
#91 merged -> available in v3.1.0 release
